themes/Tokyonight-Dark/v1/theme.json

```diff
@@ -0,0 +1,329 @@
+        "backdrop": "#191a26",
+        "textSelection": "#62a0ea"
+      },
+      "disabled": {
+        "accent": "#c0caf5",
+        "paragraph": "#c0caf5",
+        "background": "#191a26",
+        "border": "#c0caf5",
+        "heading": "#c0caf5",
+        "icon": "#c0caf5",
+        "separator": "#c0caf5",
+        "placeholder": "#c0caf5",
+        "hover": "#241f31",
+        "shade": "#241f31",
+        "backdrop": "#191a26",
+        "textSelection": "#62a0ea"
+      },
+      "selected": {
+        "accent": "#c0caf5",
+        "paragraph": "#c0caf5",
+        "background": "#191a26",
+        "border": "#c0caf5",
+        "heading": "#c0caf5",
+        "icon": "#c0caf5",
+        "separator": "#c0caf5",
+        "placeholder": "#c0caf5",
+        "hover": "#241f31",
+        "shade": "#241f31",
+        "backdrop": "#191a26",
+        "textSelection": "#62a0ea"
+      },
+      "error": {
+        "accent": "#c0caf5",
+        "paragraph": "#c0caf5",
+        "background": "#191a26",
+        "border": "#c0caf5",
+        "heading": "#c0caf5",
+        "icon": "#ed333b",
+        "separator": "#c0caf5",
+        "placeholder": "#c0caf5",
+        "hover": "#000000",
+        "shade": "#000000",
+        "backdrop": "#000000",
+        "textSelection": "#000000"
+      },
+      "success": {
+        "accent": "#4f8a10",
+        "paragraph": "#4f8a10",
+        "background": "#85ce68",
+        "border": "#e8e8e8",
+        "heading": "#4f8a10",
+        "icon": "#4f8a10",
+        "separator": "#e8e8e8",
+        "placeholder": "#a9a9a9",
+        "hover": "#000000",
+        "shade": "#000000",
+        "backdrop": "#000000",
+        "textSelection": "#000000"
+      }
+    },
+    "statusBar": {
+      "primary": {
+        "accent": "#000000",
+        "paragraph": "#c0caf5",
+        "background": "#191a26",
+        "border": "#000000",
+        "heading": "#000000",
+        "icon": "#c0caf5",
+        "separator": "#000000",
+        "placeholder": "#000000",
+        "hover": "#000000",
+        "shade": "#000000",
+        "backdrop": "#000000",
+        "textSelection": "#000000"
+      },
+      "secondary": {
+        "accent": "#000000",
+        "paragraph": "#c0caf5",
+        "background": "#191a26",
+        "border": "#000000",
+        "heading": "#000000",
+        "icon": "#c0caf5",
+        "separator": "#000000",
+        "placeholder": "#000000",
+        "hover": "#000000",
+        "shade": "#000000",
+        "backdrop": "#000000",
+        "textSelection": "#000000"
+      },
+      "disabled": {
+        "accent": "#000000",
+        "paragraph": "#000000",
+        "background": "#000000",
+        "border": "#000000",
+        "heading": "#000000",
+        "icon": "#000000",
+        "separator": "#000000",
+        "placeholder": "#000000",
+        "hover": "#000000",
+        "shade": "#000000",
+        "backdrop": "#000000",
+        "textSelection": "#000000"
+      },
```

Comment:
Is this intentional? If you are not modifying the `disabled` variant, you can just skip it. It's only required in the `base` scope.

Reply:
Actually this was not my doing at all, it seems to have placed a #000000 value in many fields without my intention. Making suggested changes now.